devops/cloud-functions/gcf-pin-ipfs-hash/index.js

```diff
@@ -37,7 +39,7 @@ const pinService = async event => {
         // interval every try. Set fields accordingly.
         let numberOfRetries = 4
         let initialRetryInterval = 500
-        let retryIntervalGrowthRate = 2
+        const retryIntervalGrowthRate = 2
```

Comment:
Nit: Perhaps this should be a file-level constant.

Reply:
Good point. Done.